src/node_sqlite.cc

```diff
@@ -1940,7 +1939,7 @@ void StatementSync::All(const FunctionCallbackInfo<Value>& args) {
     LocalVector<Name> row_keys(isolate);
 
     while ((r = sqlite3_step(stmt->statement_)) == SQLITE_ROW) {
-      if (row_keys.size() == 0) {
+      if (row_keys.empty()) {
```

Comment:
Can you undo this change since it's unrelated to the rest of the PR.

Reply:
Will do, yes.

Reply:
Removed it, thank you for reviewing the PR